assets/stylesheets/_animations.scss

```diff
@@ -1,9 +1,30 @@
-@mixin edit-post__loading-fade-animation {
-	animation: edit-post__loading-fade-animation 1.6s ease-in-out infinite;
+@mixin loading-fade-animation {
+	animation: loading-fade-animation 1.6s ease-in-out infinite;
 }
 
-@mixin edit-post__fade-in-animation($speed: 0.2s, $delay: 0s) {
-	animation: edit-post__fade-in-animation $speed ease-out $delay;
+@keyframes loading-fade-animation {
```

Comment:
We went back and forth about this.

This creates an issue where these keyframes are duplicated in all of our stylesheets because `_animations` is included in all of the style sheets. Maybe  the solution is to move it into a `animations-keyframes` stylesheet and explicitely import it in the index.scss of the modules that use it?

I don't think there's a perfect solution here.

Reply:
The other option is instead of adding these styles using SASS, add these animations to the `Animate` component and use that instead. The advantage would be that these styles would only live under that component.

Reply:
> This creates an issue where these keyframes are duplicated in all of our stylesheets because _animations is included in all of the style sheets. Maybe the solution is to move it into a animations-keyframes stylesheet and explicitely import it in the index.scss of the modules that use it?

😢 

This is the reason I could not recall when Kjell pinged me. Apologies, Kjell — this was the bit I forgot.

Reply:
> This creates an issue where these keyframes are duplicated in all of our stylesheets because _animations is included in all of the style sheets.

Heh, I missed that. 😄 That's not great. 

> Maybe the solution is to move it into a animations-keyframes stylesheet and explicitely import it in the index.scss of the modules that use it?

Yeah, I considered this as well. My main hangup was that it seems weird to decouple the keyframes from the animation mixins. Would it make more sense to _not_ include `animations.scss` everywhere, and instead, import that stylesheet explicitly wherever the animations need to be used? Right now, `animations.scss` only includes those mixins, which are useless without the keyframes anyway. 

> The other option is instead of adding these styles using SASS, add these animations to the Animate component and use that instead. The advantage would be that these styles would only live under that component.

I included couple notes on this above too. This _almost_ works, but it prevents us from using these animations as `@mixin`s, which seems a little unfortunate.

In any case, I'll keep thinking through this a bit...

Reply:
> Edit: The simplified animation names is still a big improvement though, and perhaps we can include a CSS comment why they are split out like they were, so we don't have to remember like this again?

Yeah, at the very least, I think this would be an improvement.